sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -28,8 +28,8 @@ use std::sync::Arc;
 
 /// A client to interact with a specific Azure storage blob, although that blob may not yet exist.
 pub struct BlobClient {
-    endpoint: Url,
-    client: GeneratedBlobClient,
+    pub(crate) endpoint: Url,
```

Comment:
Here and elsewhere, you might actually want to limit it to `pub(super)`, which means it's public to anything in the parent module and below.

Reply:
Done!